pkgs/applications/networking/instant-messengers/riot/riot-desktop.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv, fetchFromGitHub, yarn2nix, makeDesktopItem, electron, riot-web }:
+
+let
+  executableName = "riot-desktop";
+  version = "1.0.4";
+  riot-web-src = fetchFromGitHub {
```

Comment:
We have a [riot-web](https://github.com/NixOS/nixpkgs/blob/2d6c19efd64f416239d382cd83d27d1985b7ecb3/pkgs/applications/networking/instant-messengers/riot/riot-web.nix). Confused with why we have that or if it should be consolidated.

At any rate we could use the `${riot-web.src}` attribute.

Reply:
The `riot-web` package isn't built from source and downloads a prebuilt tarball instead, which is not usable here.

I attempted building `riot-web` from source as well, but "riot's build script does unconventional things", to quote the developers (see https://github.com/NixOS/nixpkgs/pull/57838#issuecomment-473959486)

Reply:
Sounds reasonable :+1: